src/yunohost/dyndns.py

```diff
@@ -168,9 +170,17 @@ def dyndns_subscribe(operation_logger, subscribe_host="dyndns.yunohost.org", dom
             error = "Server error, code: %s. (Message: \"%s\")" % (r.status_code, r.text)
         raise YunohostError('dyndns_registration_failed', error=error)
 
-    logger.success(m18n.n('dyndns_registered'))
+    # Yunohost regen conf will add the dyndns cron job if a private key exists
+    # in /etc/yunohost/dyndns
+    regen_conf("yunohost")
 
-    dyndns_installcron()
+    # Add some dyndns update in 2 and 4 minutes from now such that user should
+    # not have to wait 10ish minutes for the conf to propagate
+    cmd = "at -M now + {t} >/dev/null 2>&1 <<< \"/bin/bash -c 'yunohost dyndns update'\""
+    subprocess.check_call(cmd.format(t="2 min"), shell=True)
```

Comment:
I don't really understand why you do this, and why you run it two times?

Can we instead just run `yunohost dyndns update` (or directly call the function `dyndns_update` to avoid any lock problem) in regen conf?

Reply:
Yeah sorry i should have explained that in the comment

Thing is, after registering the domain on dynette, for some yet unkown reason, trying "yunohost dyndns update" right away (in the next 1~2ish minutes) won't work because it doesn't recognize the key ... that might have to do with the dynette cron (on server side) but not sure ...

So that's why i set a job a +2 and +4 minutes (because now the client cron only runs every 10 minutes)

Not sure what you mean about the lock ?

Reply:
Ok, I understand now, seems legit then.

> Not sure what you mean about the lock ?

I was talking about running yunohost command, but I was not sure if we can run yunohost command in the regenconf because of moulinette lock etc.